crates/re_sdk/src/log_sink.rs

```diff
@@ -76,6 +76,50 @@ impl LogSink for BufferedSink {
     }
 }
 
+/// The storage used by [`MemorySink`]
```

Comment:
This comment would make more sense if you declared `MemorySinkStorage` _after_ `MemorySink` 😉